### ISSUE TITLE
请问怎么调整最后的输出结果的小数点位数，目前最后的测试结果是两位，在哪里可以调整一下啊？

### ISSUE THREAD

**Issue by xinjiesun-ustc** [Tongjilibo/bert4torch#150]
提问时请尽可能提供如下信息：

### 基本信息
- 你使用的**操作系统**: 
- 你使用的**Python**版本: 
- 你使用的**Pytorch**版本: 
- 你使用的**bert4torch**版本: 
- 你加载的**预训练模型**:

### 核心代码
```python
# 请在此处贴上你的核心代码
```

### 输出信息
```shell
# 请在此处贴上你的调试输出
```

### 自我尝试
此处请贴上你的自我尝试过程

**Comment by Tongjilibo** [comment]
想问下这个小数点是指哪里的小数点啊

**Comment by Tongjilibo** [comment]
我看了下，可以在脚本最开始使用如下代码实现，默认为4
```python
import os
os.environ['ROUND_PRECISION'] = '4'
```

**Comment by xinjiesun-ustc** [comment]
- Finish Training 这个结果只有两位小数，怎么保留四位呢？？

**Comment by Tongjilibo** [comment]
> * Finish Training 这个结果只有两位小数，怎么保留四位呢？？

不好意思，没太理解你的意思，你如果说的是评估evaluate时候的指标精度，那是用户自行控制的，和框架自身无关

**Comment by xinjiesun-ustc** [comment]
比如说某数据集的训练或者测试的结果，精度都是两位  ，原始显示格式如下：          
  precision    recall  f1-score   support

           0       0.74      0.78      0.76       173
           1       0.79      0.79      0.79       346
           2       0.75      0.69      0.72       173

    accuracy                           0.76       692
   macro avg       0.76      0.75      0.76       692
weighted avg       0.76      0.76      0.76       692  ，如何在classification.py进行修改精度，我没有找到修改精度的地方，应该是我的问题，如果可以请指明一下如何修改；另外weighted avg 这个是应该是加权平均的意思，它和 accuracy的区别是？如何设置的权重呢？

**Comment by Tongjilibo** [comment]
你说的是`sklearn.metrics`中的`classification_report`函数这些吧，建议进去函数内部查一下函数的参数，比如`classification_report`有一个`digit`参数之类

**Comment by xinjiesun-ustc** [comment]
非常感谢，明白了！确实是修改digit就可以了！
